.gitlab-ci.yml

```diff
@@ -922,6 +939,41 @@ iot_agent_deb-arm64:
     paths:
       - $OMNIBUS_PACKAGE_DIR
 
+iot_agent_deb-armhf:
+  stage: package_build
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_armhf:$DATADOG_AGENT_ARMBUILDIMAGES
+  tags: ["runner:docker-arm", "platform:arm64"]
+  needs: [ "fail_on_non_triggered_tag", "build_iot_agent-deb_armhf"]
+  variables:
+    AGENT_MAJOR_VERSION: 7
+    PYTHON_RUNTIMES: '3'
+  before_script:
+    - source /root/.bashrc
+    - inv -e deps --verbose --dep-vendor-only
+    - export RELEASE_VERSION=$RELEASE_VERSION_7
```

Comment:
I also don't think this line is needed, we directly use `$RELEASE_VERSION_7` below (same for the other `arm64` IoT Agent jobs).